middle_end/flambda2/parser/print_fexpr.ml

```diff
@@ -510,6 +527,10 @@ let ternop ppf t a1 a2 a3 =
     Format.fprintf ppf "@[<2>%%array_set%a@ %a.(%a) %a %a@]"
       (array_kind ~space:Before) ak simple a1 simple a2 init_or_assign ia simple
       a3
+  | Block_set (bk, ia) ->
+    Format.fprintf ppf "@[<2>%%block_set@ %a@ %a.(%a) %a %a@]"
```

Comment:
```suggestion
    Format.fprintf ppf "@[<2>%%block_set%a@ %a.(%a)@ %a %a@]"
```
I think this is consistent with the definition of `block_access_kind` (it's already correct on line 458). (It also adds another optional break.)